src/f3dasm/_src/experimentdata/experimentsample.py

```diff
@@ -349,27 +349,26 @@ def to_dict(self) -> Dict[str, Any]:
         """
         return {**self.input_data, **self.output_data_loaded, 'job_number': self.job_number}
 
-    def store(self, object: Any, name: str, to_disk: bool = False,
+    def store(self, name: str, object: Any, to_disk: bool = False,
```

Comment:
Please clarify in the method docstring where the object is actually stored.

As far as I can understand from the documentation, the user is only providing the basename of the storage file.